cognify/optimizer/core/unified_layer_opt.py

```diff
@@ -385,7 +387,35 @@ def add_constraint(self, score, trial: optuna.trial.Trial):
             trial.set_user_attr(qc_identifier, constraint_result)
             # NOTE: add system attr at loading time
             # trial.set_system_attr(_base._CONSTRAINTS_KEY, constraint_result)
-
+    
+    def _update_best_trial(self, eval_result: EvaluationResult):
+        with self._study_lock:
+            mscore, mcost = self.get_eval_feedback(eval_result)
```

Comment:
What does the "m" before score and cost mean here?

Reply:
changed naming